services/resin-device/ssh/tcp/ssh.service

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" standalone='no'?>
+<!DOCTYPE service-group SYSTEM "avahi-service.dtd">
+<service-group>
+  <name replace-wildcards="yes">%h</name>
+  <service>
+    <type>_ssh._tcp</type>
+    <subtype>_resin-device._sub._ssh._tcp</subtype>
+    <port>2222</port>
```

Comment:
Nit (since I don't think this is used during service enumeration, does it) - the device SSH port is `22222`